js/src/common/classes/index.js

```diff
@@ -0,0 +1 @@
+export { default as GAClient } from './GAClient';
```

Comment:
This is silly for one file, but it follows the pattern of exporting files in all of the other directories